### ISSUE TITLE
Won't work initially when starting Chrome

### ISSUE THREAD

**Issue by enewshound** [mikecrittenden/shortkeys#254]
When I start Chrome, Shortkeys does not function initially.  But it does work perfectly after I refresh my startup page, my Home Page.  Tried everything I know how to do for days with no luck

## Your Environment
Shortkeys version 2.2.7
Version 74.0.3729.169 (Official Build) (64-bit)
Windows 10

**Comment by mikecrittenden** [comment]
Closing some old issues, feel free to comment if this is still an issue.

**Comment by enewshound** [comment]
Yes, it is still very much an issue.  Have tried every fix in the book.  But it still does this about two-thirds of the times when I start Chrome.  The only fix that works is to reload the page.  I have two pages that open each time I open Chrome and they stay opened, and both of them exhibit this.  One is Gmail and the other page is my personal home page.  If I sart Chrome, Shortkeys won't work without a reboot.  Then if I go to the home page, it still won't work there until I reload that page.  Then it stays working fine from then on.    I have reinstalled Shortkeys several time with no change in the problem.  Help!

**Comment by mikecrittenden** [comment]
Thanks! I'm out for a few days for the holidays but I'll do some research here.

**Comment by enewshound** [comment]
Thank you, Mike.  As I opened Chrome just now, it did it again!

Happy holidays.

---

 John Marshall

Phone or Text:  213-220-3006

**Comment by enewshound** [comment]
One more thing, Mike.  Shortkeys works perfectly in Firefox.  But Chrome
requires a reload of the current page before Shortkeys kicks in and works.

BTW, I absolutely love the app even with the minor glitch.  I am on the
computer several hours a day, and I constantly use Shortkeys to get around
in it.

Thank you.

---

 John Marshall

Phone or Text:  213-220-3006

**Comment by mikecrittenden** [comment]
This is an odd one. I can't reproduce it, and I've never seen anyone else report this issue. Do you by any chance have any local extensions or settings that could be caching things more aggressively for you? Or any script blockers, etc.? Any other extensions or security settings you can think of that might be affecting this?

**Comment by enewshound** [comment]
Thanks, Mike. I tried turning off every extension in my Chrome browser, but
no effect. I have spent weeks experimenting with all the settings and
blockers, but again found nothing. Eventually. I will stumble onto it by
accident whatever it might be.

Again, Mike, thank you for your time and effort to help. I certainly love
the app otherwise. Keep up the good work!

John Marshall
Call or text 213-220-3006
